flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -9,6 +9,7 @@ class SentryFlutterOptions extends SentryOptions {
   SentryFlutterOptions({String? dsn, PlatformChecker? checker})
       : super(dsn: dsn, checker: checker) {
     enableBreadcrumbTrackingForCurrentPlatform();
+    enablePrintBreadcrumbs = false;
```

Comment:
is that the only option? I'd prefer rather a workaround if possible :D

Reply:
We could intercept https://github.com/flutter/flutter/blob/f4abaa0735eba4dfd8f33f73363911d63931fe03/packages/flutter/lib/src/foundation/assertions.dart#L919

and disable recording when it's called.
Kinda like what we do it for FlutterError.onError.
Does that sound better?

Reply:
sounds better to me, rather than disabling the feature by default.
we can check if its the `FlutterExceptionHandler` type and do not log it if its a release build, because, on debug level, printing to console is rather useful